lib/src/e2ee/e2ee_manager.dart

```diff
@@ -117,6 +121,11 @@ class E2EEManager {
             }
           }
         });
+      if (_encryptionEnabled && room.roomOptions.encryption != null) {
+        _dataPacketCryptor ??=
```

Comment:
I think the problem with this is that it only sets up the dataPacketCryptor if `roomOptions.encryption` is set. 

However the behaviour we'd want is to support _de_cryption also when `e2ee` has been set on the room options instead of `encryption`. 
If `encryption` is set, then also _en_cryption for data channels will be performed, but we'll need to make sure that the decryption path works for both cases.